### PR TITLE
Remove elrond-go dependency

### PR DESCRIPTION
**Pull request by miiu96**
- Removed elrond-go dependency
- Updated elrond-go-core module
- Updated elrong-go-logger module

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,10 +4,9 @@ go 1.17
 
 require (
 	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
-	github.com/ElrondNetwork/elrond-go v1.3.27
-	github.com/ElrondNetwork/elrond-go-core v1.1.18
+	github.com/ElrondNetwork/elrond-go-core v1.1.21-0.20220928110648-e602417daa59
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
-	github.com/ElrondNetwork/elrond-go-logger v1.0.7
+	github.com/ElrondNetwork/elrond-go-logger v1.0.8-0.20221003085013-fb326ba1620f
```
Comment (bogdan-rosianu) [inline go.mod:9]: reminder to use proper tags before merging

File: process/nodeStatusProcessor.go [hunk 2]
```diff
@@ -47,6 +46,12 @@ const (
 
 	// EnableEpochsPath represents the path where an observer exposes all the activation epochs
 	EnableEpochsPath = "/network/enable-epochs"
+
+	// MetricCrossCheckBlockHeight is the metric that store cross block height
```
Comment (bogdan-rosianu) [inline process/nodeStatusProcessor.go:50]: stores

File: go.mod [hunk 3]
```diff
@@ -4,10 +4,9 @@ go 1.17
 
 require (
 	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
-	github.com/ElrondNetwork/elrond-go v1.3.27
-	github.com/ElrondNetwork/elrond-go-core v1.1.18
+	github.com/ElrondNetwork/elrond-go-core v1.1.21-0.20220928110648-e602417daa59
```
Comment (iulianpascalau) [inline go.mod:7]: do not forget about proper releases

File: process/blocksProcessor_test.go [hunk 4]
```diff
@@ -2,9 +2,10 @@ package process_test
 
 import (
 	"errors"
-	"github.com/ElrondNetwork/elrond-go-core/data/api"
+
```
Comment (iulianpascalau) [inline process/blocksProcessor_test.go:5]: extra space

Comment (miiu96) [reply process/blocksProcessor_test.go]: done

File: go.mod (hunk 1 above)
Comment (miiu96) [reply go.mod]: done

File: process/nodeStatusProcessor.go (hunk 2 above)
Comment (miiu96) [reply process/nodeStatusProcessor.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**